src/Dfe.PlanTech.Web/Controllers/QuestionsController.cs

```diff
@@ -93,8 +94,18 @@ public async Task<IActionResult> SubmitAnswer(SubmitAnswerDto submitAnswerDto, [
             });
             return RedirectToAction("GetQuestionById");
         }
+
+        string? nextQuestionId;
+        try
+        {
+            nextQuestionId = await submitAnswerCommand.GetNextQuestionId(submitAnswerDto.QuestionId, submitAnswerDto.ChosenAnswerId);
+        }
+        catch (Exception e)
+        {
+            logger.LogError( "An error has occurred while retrieving the next question with the following message: {} ", e.Message);
+            throw new ContentfulDataUnavailableException($"An error has occurred while retrieving the next question with the following message: {e.Message}", e);
```

Comment:
i don't like the exception being thrown on a caught exception.

Move the try catch to get the `GetNextQuestionId` method IMO